pkg/lib/configreader/validators.go

```diff
@@ -149,3 +150,26 @@ func DurationParser(v *DurationValidation) func(string) (interface{}, error) {
 		return d, nil
 	}
 }
+
+func ValidateImageVersion(image string) (string, error) {
```

Comment:
Yeah, I think that makes sense. I think that's okay.

Reply:
That's fine to me. One thing to keep in mind: I had to create a wrapper function for `ValidateImageVersion(image, version string)` inside `clusterconfig/clusterconfig.go` called `validateImageVersion(image string)` because this function is still required as a validator to other cluster-configured images and because the validator expects a single parameter.